src/plugins/plot/stackedPlot/StackedPlotItem.vue

```diff
@@ -120,7 +120,9 @@ export default {
             });
         },
         onTickWidthChange() {
-            this.$emit('plotTickWidth', ...arguments);
+            const id = this.openmct.objects.makeKeyString(this.object.identifier);
+            const args = [...arguments, id];
+            this.$emit('plotTickWidth', ...args);
```

Comment:
probably either get all arguments from `mctplot.vue`, or only accept tick width from mctplot.vue and add id